data/json/items/comestibles/carnivore.json

```diff
@@ -661,7 +698,25 @@
     "parasites": 0,
     "healthy": -2,
     "fun": -8,
-    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+    "extend": { "flags": [ "EATEN_HOT" ] }
```

Comment:
Is that a on purpose? Brain is now nice eaten hot?

Reply:
I figured it'd be nicer hot than cold, though I don't have any first-hand experience.

Reply:
I've eated brain mutiple times, it is good when cooked appropriately.
I only tried it cold, as a "pâté", but I think the "beef brain" dishe ( calf brain for what I'm aware ) is eated hot.
I don't know why it is unhealthy in the game, prions being the only problem associated with eating brain I'm aware of. ( and prions are rare outbreaks )

![beef brain nutrition sheet](https://www.doctorkiltz.com/beef-brain/)

Reply:
Oh *come on*, why are organ meats universally negative health? Yeah, that gets axed. Weird mutant bits being unhealthy is fair game, so normal organs get a slight positive and mutated ones keep the slight negative values.